examples/quickstart.rs

```diff
@@ -0,0 +1,62 @@
+use eventstore::{
+    AppendToStreamOptions, Client, Credentials, EventData, ExpectedVersion, ReadResult,
+    ReadStreamOptions,
+};
+use futures::TryStreamExt;
+use serde::{Deserialize, Serialize};
+use std::error::Error;
+use uuid::Uuid;
+
+#[derive(Serialize, Deserialize)]
+struct TestEvent {
+    pub id: String,
+    pub important_data: String,
+}
+
+type Result<A> = std::result::Result<A, Box<dyn Error>>;
+
+pub async fn run() -> Result<()> {
+    // region createClient
+    let settings = "{connectionString}".parse()?;
+    let client = Client::create(settings).await?;
```

Comment:
**SUGGESTION:** We discussed that internally, I think that using `EventStoreDBClient` would be more explicit. I agree that [use](https://github.com/EventStore/EventStoreDB-Client-Rust/pull/49/files#diff-f7fe53d6c43565fb44b326fcb51cfeab13efb52080e96701335c8a6587500abcR1) will look a bit weird, but usage will be more explicit. Especially that generic `Client` name can easily conflict with other class named like that.
(it's not a blocker for me).

Reply:
Not possible in Rust because the module system is different. There is no wildcard import by default. I would advise us to keep `Client` instead. I'll do extra research to see what other database clients do.